pages/mint/index.tsx

```diff
@@ -0,0 +1,5 @@
+import MintPage from "../../components/MintPage"
+
+const Mint = () => <MintPage type="mint" />
```

Comment:
can you move `<MintProvider />` here?

or, inside the `<MintPage />` component?